lib/chef/application.rb

```diff
@@ -60,10 +61,11 @@ def reconfigure
     end
 
     # Get this party started
-    def run
+    def run(enforce_license = false)
```

Comment:
It would be nice to do this as an options hash so it's more apparent what the true we're passing into new is up in the binaries. Right now you have to hop down here to see what is being set.